src/schema/v2/artwork/index.ts

```diff
@@ -943,6 +949,55 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return savedArtworkLoader(_id).then(({ is_saved }) => is_saved)
         },
       },
+      isSavedToList: {
+        description: "Checks if artwork is saved to user's lists",
+        args: {
+          default: {
+            type: GraphQLBoolean,
+            defaultValue: false,
+          },
+          saves: {
+            type: GraphQLBoolean,
+            defaultValue: true,
+          },
+        },
+        type: new GraphQLNonNull(GraphQLBoolean),
+        resolve: async (
+          { _id },
+          args,
+          { savedArtworkLoader, collectionsLoader, userID }
+        ) => {
+          try {
+            const { default: isDefault, saves } = args
+
+            // For the default saved artworks list, we can use a different loader
+            // that is more performant.
+            if (isDefault && saves) {
+              if (!savedArtworkLoader) return false
+
+              const { is_saved: isSaved } = await savedArtworkLoader(_id)
+              return isSaved
+            }
+
+            if (!collectionsLoader || !userID) return null
```

Comment:
Am I reading this right? This returns `null`, but is declared non-nullable on L964?

Reply:
Oh yes good call! Whoops, meant to do `return false`.